extractor/filesystem/os/bazelmaven/metadata/metadata.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package metadata defines a metadata struct for Bazel Tools.
+package metadata
+
+import (
+	pb "github.com/google/osv-scalibr/binary/proto/scan_result_go_proto"
+)
+
+// Metadata holds parsing information for a Bazel tool.
```

Comment:
This is mostly the same as https://github.com/google/osv-scalibr/blob/main/extractor/filesystem/language/java/javalockfile/metadata.go so we could reuse that instead - unless you're planning to add more fields functions like rule names as discussed further up.